### ISSUE TITLE
basic.get is broken

### ISSUE THREAD

**Issue by wavetossed** [agoragames/haigha#6]
I have a simple test program that is basically the same as the example on PyPi except that it uses basic.get and no_ack = False. Also the queues are durable=True and I have a test queue with a few messages in it. When I use basic.consume it all works fine and I can run it again and again. Change to basic.get and it craps out with the exact same messages (unacked in durable queue).

In classes/basic_class.py I changed this function

 def _recv_get_ok(self, method_frame):
    msg = self._read_gokmsg( method_frame )
    cb = self._get_cb.pop()
    if cb: cb( msg )

_read_gokmsg is the same as _read_msg except for this:

```
delivery_tag = method_frame.args.read_longlong()
redelivered = method_frame.args.read_bit()
exchange = method_frame.args.read_shortstr()
routing_key = method_frame.args.read_shortstr()
message_count = method_frame.args.read_long()

delivery_info = {
  'channel': self.channel,
  'delivery_tag': delivery_tag,
  'redelivered': redelivered,
  'exchange': exchange,
  'routing_key': routing_key,
  'message_count': message_count,
}
```

Basically I removed the consumer tag and added the message count. Now I can use basic.get just fine.

P.S. your code is an awful lot easier to dig into than kombu or amqplib
